indy-besu/smart_contracts/contracts/did/DidRegistry.sol

```diff
@@ -48,14 +57,17 @@ contract DidRegistry is DidRegistryInterface, ControlledUpgradeable {
         DidValidator.validateVerificationKey(document);
 
         _dids[document.id].document = document;
+        _dids[document.id].metadata.creatorAddress = msg.sender;
         _dids[document.id].metadata.created = block.timestamp;
         _dids[document.id].metadata.updated = block.timestamp;
 
         emit DIDCreated(document.id);
     }
 
     /// @inheritdoc DidRegistryInterface
-    function updateDid(DidDocument calldata document) public _didExist(document.id) _didIsActive(document.id) {
+    function updateDid(
+        DidDocument calldata document
+    ) public _didExist(document.id) _didIsActive(document.id) _calledByCreator(document.id) {
```

Comment:
Can you please update the documentation for the `updateDid` and `deactivateDid` methods in `DidRegistryInterface`, as there are new restriction added?

Reply:
Added docs